### ISSUE TITLE
Multiselect

### ISSUE THREAD

**Issue by Semsveta** [primefaces/primevue#8010]
### Describe the bug

The MultiSelect component has multiple accessibility violations. Please refer to this Storybook example, where the violations are listed in the Accessibility tab: https://semsveta.github.io/storybook-issue/?path=/story/multiselect--default

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-jcmtragd?file=src%2FApp.vue

### Environment

Accessibility Testing Tool: 
Example: axe-core, Storybook Accessibility Addon

### Vue version

3.5.17

### PrimeVue version

4.3.6

### Node version

20.18.0

### Browser(s)

Chrome 138.0.7204.169

### Steps to reproduce the behavior

1. Please go to git pages: https://semsveta.github.io/storybook-issue/?path=/story/multiselect--default
2. Check the Accessibility tab to see issues listed

### Expected behavior

Expected behavior: at least critical violations 'Form label' : 'aria-label attribute does not exist or is empty'.
'aria-labelledby attribute does not exist' should not be present.

**Comment by Semsveta** [comment]
There is also an accessibility error: "Interactive controls must not be nested", which occurs when using an input of type checkbox inside an unordered list for the options. This limitation is documented in the [WAI-ARIA Listbox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/), which states:
"...the interaction model conveyed by the listbox role to assistive technologies does not support interacting with elements inside of an option. Because of these traits of the listbox widget, it does not provide an accessible way to present a list of interactive elements, such as links, buttons, or checkboxes."
To resolve this issue, I had to write JavaScript code that manipulates the DOM to replace the input elements with divs. However, this approach is not good, especially when dealing with a large number of options, as it can lead to performance issues. Would be grate if you can fix it on your side.

I also wanted to mention an accessibility issue related to the use of the aria-controls attribute in several components that include popups. Specifically, aria-controls is often set to reference the id of an element that does not exist in the DOM until the popup is activated. This causes the following accessibility error:
Unable to determine if aria-controls referenced ID exists on the page while using aria-haspopup.

**Comment by manankharwar** [comment]
For MultiSelect accessibility improvements, AccessOps can test keyboard navigation and screen reader compatibility systematically. Works well with Vue components for ongoing accessibility testing. https://accessops.dev/
